modules/m_whois.c

```diff
@@ -373,10 +375,20 @@ single_whois(struct Client *source_p, struct Client *target_p, int operspy)
 					target_p->name, buf);
 		}
 
+		/* fire the doing_whois_show_idle hook to allow modules to tell us whether to show the idle time */
+		hook_data_client_approval hdata_showidle;
+
+		hdata_showidle.client = source_p;
+		hdata_showidle.target = target_p;
+		hdata_showidle.approved = 1;
+
+		call_hook(doing_whois_show_idle_hook, &hdata_showidle);
+
+		/* operspies are always allowed to see idle time */
```

Comment:
Removing this comment to go along with my comment about not lumping operspy in with this.

```suggestion
```